packages/react-components/src/components/RichTextEditor/Plugins/CopyPastePlugin.tsx

```diff
@@ -50,6 +53,46 @@ const handleBeforePasteEvent = (
   }
 };
 
+/* @conditional-compile-remove(rich-text-editor-image-upload) */
+/**
+ * @internal
+ * Exported only for unit testing
+ */
+export const handleInlineImage = (
+  event: PluginEvent,
+  onUploadImage?: (image: string, fileName: string) => void
+): void => {
+  // We don't support the pasting options such as paste as image yet.
+  if (event.eventType === PluginEventType.BeforePaste && event.pasteType === 'normal') {
```

Comment:
We probably can add a check here (in `if`) that `onUploadImage` exists as we don't need to do any processing otherwise